examples/quick-start-artillery/.env.template

```diff
@@ -0,0 +1 @@
+TRACETEST_AGENT_API_KEY=
```

Comment:
Would it be possible to add the `TRACETEST_API_TOKEN` here as well and load it in the yaml files? This could mean removing the hardcoded token values from the yamls and making sure the user only sets it once.